src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx

```diff
@@ -28,6 +29,7 @@ export type Props = {
     options: Pick<SingleIntegrationEvent, 'eventKey' | 'eventName'>
   ) => void; //analytics callback
   className?: string;
+  showSlackAlert?: boolean;
```

Comment:
Also don't really like this highly specific type, but also 🤷‍♂️

Reply:
@EvanPurkhiser Yea, I feel you. I'd be down to generalize it if we ever have another use of it.